codec-server/protocol.md

```diff
@@ -0,0 +1,66 @@
+# Codec Server: protocol
```

Comment:
I think these details can go right in the README, but if you don't want to, may at least want to link from there.

Reply:
sg, moved.